### PR TITLE
Rich text editor

### PR DESCRIPTION
**Pull request by Jay-Gadhiya**
Test Link: https://cheer-notes-ij9a2b271-jay-gadhiya.vercel.app/

Feature add: rich text editor

### Future Features
- I will refactor css in this
- add more options in a rich text editor

### CODE REVIEW

File: src/Contexts/notesActions-context.jsx [hunk 1]
```diff
@@ -13,11 +13,12 @@ const NotesProvider = ({ children }) => {
         content : "", 
         date : "", 
         flag : false, 
-        id : "",
+        _id : "",
         color : ""
     });
     const [activePage, setActivePage] = useState("notes");
     const [color, setColor] = useState("");
+    console.log(userNote);
```
Comment (Sudhanshusoni45) [inline src/Contexts/notesActions-context.jsx]: You can avoid having unnecessary logs in a pr .

File: src/Pages/Home/Home.jsx [hunk 2]
```diff
@@ -11,15 +11,17 @@ import { useAuth } from "../../Contexts/authentication-context";
 import { addNote } from "../../Utilities-Functions/addNote";
 import { editNote } from "../../Utilities-Functions/editNote";
```
Comment (Sudhanshusoni45) [inline src/Pages/Home/Home.jsx:12]: You can export/import all files of a folder from a single file index.js

**Review by Sudhanshusoni45 — APPROVED** [review]
1.Nice Ui
2.Well written code

Suggestion: you can increase the width of your Note Card seems little small.

**Review by RahullRawat — APPROVED** [review]
**Your code is looking good and functionality is working fine.**

You can remove all console.log and commented code

**Pull request merged**